<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App Creator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>791</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_control">
     <attribute name="title">
      <string>control</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_control">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>771</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_desktop">
     <attribute name="title">
      <string>.desktop</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_desktop">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>781</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_info">
     <attribute name="title">
      <string>info</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>761</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>其他</string>
     </attribute>
     <widget class="QLabel" name="label_developer">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>401</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_snapdownurl">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>751</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_snappkg">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>401</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_selectdeb">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>531</width>
      <height>36</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_selectdeb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string> 选择deb包</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_searchsnap">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>121</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索snap包</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_selectoutpath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>106</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>选择输出路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_outpath">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>610</y>
      <width>661</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_convert">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>690</y>
      <width>106</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>661</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>功能：1、现有deb-&gt;uos         2、snap-&gt;deb，snap包只需输入名称即可查询，优先显示x86，可自定义</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string> 版本：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_version">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_x86">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>x86</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_arm">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>arm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pkgname">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>511</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_pkgname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>包名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_appname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>应用名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_appname">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>511</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_snapname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>201</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_debname">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>650</y>
      <width>661</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_size1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Size ：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_size2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>生成的deb名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_arch2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>架构：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_arch">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
